language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/SnippetGenerator.java

```diff
@@ -1569,7 +1569,7 @@ public static SnippetBlock getRemoteMethodDeclSnippet() {
      * @return {@link SnippetBlock}     Generated Snippet Block
      */
     public static SnippetBlock getInitFunctionSnippet() {
-        String snippet = "function init(${1:any arg}) {" + CommonUtil.LINE_SEPARATOR + "\t${2}" +
+        String snippet = "function init() {" + CommonUtil.LINE_SEPARATOR + "\t${2}" +
```

Comment:
Shouldn't we provide tab navigation for the user to define parameters  in the case of a class definition?